examples/azure/build-and-deploy-multiple-artifacts.yml

```diff
@@ -0,0 +1,184 @@
+  pool:
+    vmImage: ubuntu-latest
+  # conrainer with pre-installed mender-artifact and mender-cli tools
+  container: mendersoftware/mender-ci-tools:mender-master
+  steps:
+  # Creates Mender Artifact, using randomly generated dummy name
+  # In this example, hello-mender.py represents a single file Python application that we have
+  # built and test in our CI and now we want to deploy into a device running Mender. We will
+  # package it as a Single File Application Update. For more information visit Mender docs:
+  # https://docs.mender.io/artifact-creation/create-an-artifact
+  - script: |
+      curl -O https://raw.githubusercontent.com/mendersoftware/mender-ci-workflows/master/examples/hello-mender.py
+      chmod 755 hello-mender.py
+      single-file-artifact-gen \
+        --device-type raspberrypi4 \
+        -o ${MENDER_ARTIFACT_NAME_TWO} \
+        -n ${MENDER_RELEASE_NAME_TWO} \
+        --software-name hello-mender \
+        --software-version 1.0 \
+        --dest-dir /usr/local/bin \
+        hello-mender.py
+    displayName: Build Mender Artifact
+  # Uploads Azure DevOps artifacts to Azure artifact storage to pass them to the further jobs
+  - task: PublishPipelineArtifact@1
+    inputs:
+      targetPath: $(MENDER_ARTIFACT_NAME_TWO)
+      artifactName: mender_two
+
+
+#
+# Uploads built Mender Artifacts to Mender server
+#
+- job: upload_mender_artifact_one
+  dependsOn:
+  - build_mender_artifact_one
+  pool:
+    vmImage: ubuntu-latest
+  steps:
+  # Downloads built in previous job Azure DevOps artifacts
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      artifact: mender_one
+      patterns: $(MENDER_ARTIFACT_NAME_ONE)
+      path: $(System.DefaultWorkingDirectory)
+  # Uploads Mender Artifact to a Mender server
+  - template: templates/azure/mender-artifact-upload.yml@templates
+    parameters:
+      mender_uri: $(MENDER_SERVER_URL)
+      mender_pat: $(MENDER_SERVER_ACCESS_TOKEN)
+      mender_artifact: $(MENDER_ARTIFACT_NAME_ONE)
+
+- job: upload_mender_artifact_two
+  dependsOn:
+  - build_mender_artifact_two
+  pool:
+    vmImage: ubuntu-latest
+  steps:
+  # Downloads built in previous job Azure DevOps artifacts
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      artifact: mender_two
+      patterns: $(MENDER_ARTIFACT_NAME_TWO)
+      path: $(System.DefaultWorkingDirectory)
+  # Uploads Mender Artifact to a Mender server
+  - template: templates/azure/mender-artifact-upload.yml@templates
+    parameters:
+      mender_uri: $(MENDER_SERVER_URL)
+      mender_pat: $(MENDER_SERVER_ACCESS_TOKEN)
+      mender_artifact: $(MENDER_ARTIFACT_NAME_TWO)
+
+#
+# Creates deployments on Mender server
+#
+- job: create_deployment_one
+  dependsOn:
+  - upload_mender_artifact_one
+  pool:
+    vmImage: ubuntu-latest
+  steps:
+  # Sets Mender deployment name
+  - script: |
+      echo "##vso[task.setvariable variable=MENDER_DEPLOYMENT_NAME;isOutput=true;]deployment_one"
+    name: set_variable
```

Comment:
What is this? :exploding_head:  A reference to the documentation would help here.